drivers/cc110x/cc110x_settings.c

```diff
@@ -20,7 +20,7 @@
 #include "cc110x.h"
 #include "cc110x_internal.h"
 
-const char cc110x_conf[CC110X_CONF_SIZE] = {
+const unsigned char cc110x_conf[CC110X_CONF_SIZE] = {
```

Comment:
Why not use `uint8_t` instead of `unsigned char` ? It's standard C99 type.

Reply:
Can do, just thought since it was `char` before this would keep with that style.

Reply:
Feel free to use `uint8_t` ;-)

Reply:
Alright, shouldn't make a difference.